content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -445,6 +445,9 @@ and
 [kubectl apply](/docs/reference/generated/kubectl/kubectl-commands/#apply).
 
 
+{{< note >}}
+Strategic merge patch is not supported for {{< glossary_tooltip term_id="CustomResourceDefinition" text="custom resources" >}}.
```

Comment:
According to the discussion and the upstream issue, strategic merge patch is not supported for custom resources.
I'm wondering if strategic merge patch is supported for CRDs, I believe so.
If that is the case, the tooltip is wrong.

Reply:
the tooltip is a link to CRDs, and the text is `custom resources`

The tool tip is added by @sftim and I thinks it makes sense for users to learn more about what is custom resource and CRD as well. I accept it as this is reasonable. (https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#create-custom-objects)

Reply:
The fact is that strategic merge patch is not supported for custom resources due to implementation constraints, i.e. no GoLang tag to capture patch strategy or merge keys. I'm not sure we can add this support by adding `x-kubernetes-patch-strategy` to the CRD schema. Anyway ... linking this to CustomResourceDefinition which is itself a builtin "resource" is not appropriate.

Reply:
Let me test it later.  After testing, I will get back to this.

BTW,  your suggestions would be `remove the tooltip` if this is needed?

Reply:
My suggestion is to put as simple/straightforward as it could be:

> Strategic merge patch is not supported for custom resources.